### PR TITLE
Test realm

### PR DESCRIPTION
**Pull request by mdellweg**
Finally managed to get this running.
And it shows, that realm is _not_ idempotent.

### CODE REVIEW

File: plugins/modules/foreman_realm.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def main():
             server_url=dict(required=True),
             username=dict(required=True, no_log=True),
             password=dict(required=True, no_log=True),
-            verify_ssl=dict(type='bool', default=True),
+            validate_certs=dict(type='bool', default=True),
```
Comment (evgeni) [inline plugins/modules/foreman_realm.py]: alias. plz.

File: plugins/modules/foreman_realm.py [hunk 2]
```diff
@@ -139,15 +139,15 @@ def main():
...
+    verify_ssl = realm_dict.pop('validate_certs')
     state = realm_dict.pop('state')
 
     try:
         create_server(server_url, (username, password), verify_ssl)
     except Exception as e:
         module.fail_json(msg="Failed to connect to Foreman server: %s " % e)
 
-    ping_server(module)
+    # ping_server(module)
```
Comment (evgeni) [inline plugins/modules/foreman_realm.py]: cheater (but fair)

File: plugins/modules/foreman_realm.py (hunk 1 above)
Comment (evgeni) [reply plugins/modules/foreman_realm.py]: (or don't do the change, up to you)

**Comment by evgeni** [comment]
Thank you @mdellweg!

Do you want to make it idempotent in this PR? Or should we ignore it for now?
If not, I'd just merge it as-is, because it is a huge improvement already.

@claudiol do you want to review this PR, which updates your #85?

**Comment by mdellweg** [comment]
I decided to do as little change to the module as necessary. And then i just rewrote it for apypie.
#384

**Comment by evgeni** [comment]
I think that's fair. Thank you!

**Pull request merged**